res/values/strings.xml

```diff
@@ -301,27 +301,27 @@
 
     <!-- welcome and login -->
     <string name="welcome_intro1_headline">Delta Chat</string>
-    <string name="welcome_intro1_message">The messenger with the broadest audience in the world. Free and independent.</string>
```

Comment:
i think, we had this discussion free - libre already at some point, please just leave them for now. but there is a broader discussion at https://github.com/deltachat/interface/issues/25